Source/UI/CommonUI/Protocols/Elements/ElementBorderStyle.swift

```diff
@@ -1,11 +1,11 @@
 import UIKit
 
-public protocol ElementBorderStyle {
-    var cornerRadius: CGFloat { get }
-    var borderWidth: CGFloat { get }
-    var normalColor: UIColor { get }
-    var focusColor: UIColor { get }
-    var errorColor: UIColor { get }
-    var edges: UIRectEdge? { get }
-    var corners: UIRectCorner? { get }
-}
+    public protocol ElementBorderStyle {
+        var cornerRadius: CGFloat { get }
+        var borderWidth: CGFloat { get }
+        var normalColor: UIColor { get }
+        var focusColor: UIColor { get }
+        var errorColor: UIColor { get }
+        var edges: UIRectEdge? { get }
+        var corners: UIRectCorner? { get }
+    }
```

Comment:
There doesn't appear to be anything else inside this file, so why the indentation?